CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.54.10] - 2024-07-26,
+### Fixed
+- Creating a Group with an `UnknownAcl` supported by the API no longer raises a `ValueError`.
```

Comment:
One typo + suggest we add in info that the create call still went through previously
```suggestion
## [7.54.10] - 2024-07-26
### Fixed
- Creating a Group with an `UnknownAcl` supported by the API no longer raises a client-side `ValueError` after
  successfully creating the group.
```